### ISSUE TITLE
Gutenboarding: Fix content spacing on each page

### ISSUE THREAD

**Issue by dubielzyk** [Automattic/wp-calypso#40605]
Currently the spacing between the top bar, the heading+text and the content is a bit off. 

Here's the correct measurements:
![image](https://user-images.githubusercontent.com/2896062/78019366-e502d480-734f-11ea-8e7c-f9fe0aef2d52.png)

(Ignore the 72px as that will be dynamic depending on how many lines of text appear).

On mobile we should use these specs:
![image](https://user-images.githubusercontent.com/2896062/78019451-0237a300-7350-11ea-889f-97e4b9214c12.png)

(Again, ignore the 105px here).

**In summary**
Spacing between the top bar and the headline: `desktop: 64px` `mobile: 44px`
Spacing between the text and the content: `desktop: 64px` `mobile: 44px`

Let's make sure we have this for all the pages that follow the same format (design, style, plans, purchase, domain).
